ember/app/adapters/application.js

```diff
@@ -0,0 +1,26 @@
+import JSONAPIAdapter from "@ember-data/adapter/json-api";
+
+export default class ApplicationAdapter extends JSONAPIAdapter {
+  namespace = "api/v1";
+
+  _appendInclude(url, adapterOptions) {
+    if (adapterOptions?.include) {
+      return `${url}?include=${adapterOptions.include}`;
+    }
+    return url;
+  }
+
+  urlForUpdateRecord(id, modelName, snapshot) {
+    return this._appendInclude(
+      super.urlForUpdateRecord(id, modelName, snapshot),
+      snapshot.adapterOptions
+    );
+  }
+
+  urlForCreateRecord(modelName, snapshot) {
+    return this._appendInclude(
+      super.urlForCreateRecord(modelName, snapshot),
+      snapshot.adapterOptions
+    );
+  }
```

Comment:
I'd like a little comment explaining why you need this here. If I read it correctly, it's because you want `include` support for update / create, right?